handsontable/handsontable
# Cancelled changes shouldn't call validation.

### Context
Full context is described by @jdmcnair in #5557 - it's PR to #3381.
It's just code clean up and small fix with a no-closing editor.

### How has this been tested?
1. Add following options to init config
```
beforeChange: () => false;
afterValidate: () => alert('afterValidate');
```
2. Open editor (eg.: double-click on cell)
3. Click on another cell
You should be able to select another cell without any alert.

### Types of changes

### Related issue(s):
1. #3381
